action.yml

```diff
@@ -2,10 +2,10 @@ name: 'Twilio SMS'
 author: 'Twilio Labs'
 description: 'Send an SMS from GitHub Actions using Twilio Programmable SMS'
 inputs:
-  fromPhoneNumber:
+  FROM_PHONE_NUMBER:
     description: 'Phone number in your Twilio account to send the SMS from'
     required: true
-  toPhoneNumber:
+  TO_PHONE_NUMBER:
```

Comment:
you should check the README PR since I think it refers to `fromPhoneNumber` and `toPhoneNumber`.